src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LanguageClient/ProjectSystem/RazorProjectInfoEndpointPublisher.TestAccessor.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+using Microsoft.VisualStudio.LanguageServer.ContainedLanguage;
+
+namespace Microsoft.VisualStudio.Razor.LanguageClient.ProjectSystem;
+
+internal partial class RazorProjectInfoEndpointPublisher
+{
+    internal TestAccessor GetTestAccessor() => new(this);
+
+    [SuppressMessage("RoslynDiagnosticsMaintainability", "RS0043:Do not call 'GetTestAccessor()'", Justification = "Used by test code only")]
+    internal static TestAccessor CreateTestInstanceAccessor(LSPRequestInvoker requestInvoker, IProjectSnapshotManager projectSnapshotManager)
```

Comment:
Please don't suppress this message -- just fix it! :smile: The method should be called "GetTestAccessor()" like all other implementations of this pattern in the code base.

Reply:
Renamed, thanks. It was pretty unclear that the name of *this* method was the issue.